state/metadata/state_manager_test.go

```diff
@@ -0,0 +1,82 @@
+// Copyright (C) 2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package metadata
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestCompatiblePrefixes(t *testing.T) {
+	require := require.New(t)
+	metadataManager := NewDefaultManager()
+
+	require.False(
+		HasConflictingPrefixes(
+			metadataManager,
+			[][]byte{
+				{DefaultMinimumPrefix},
+			},
+		),
+	)
+}
```

Comment:
Can we move this to be a test case under the other test by adding a boolean `hasConflict` to the test struct?